erpnext/manufacturing/doctype/job_card/job_card.js

```diff
@@ -32,6 +42,16 @@ frappe.ui.form.on("Job Card", {
 		});
 	},
 
+	setCompanyFilters(frm, fieldname) {
```

Comment:
Please use the proper naming convention, we don't use camelCase in function names.